tests/unit/NameRequestInfo.spec.ts

```diff
@@ -12,34 +12,34 @@ const vuetify = new Vuetify({})
 const store = getVuexStore()
 document.body.setAttribute('data-app', 'true')
 
-describe('Name Request Info component', () => {
-  let wrapper: any
-
-  const mockNrData = {
-    nrNumber: 'NR 1234567',
-    entityType: 'BEN',
-    filingId: null,
-    applicant: {
-      addressLine1: '45 Frasier Drive',
-      addressLine2: null,
-      addressLine3: null,
-      city: 'Victoria',
-      countryTypeCode: 'CA',
-      postalCode: 'V9E 2A1',
-      stateProvinceCode: 'BC',
-      emailAddress: 'test@gov.bc.ca',
-      phoneNumber: '250-356-9090',
-      firstName: 'John',
-      middleName: 'Joe',
-      lastName: 'Doe'
-    },
-    details: {
-      approvedName: 'MADRONA BREAD BASKET INC.',
-      consentFlag: null,
-      expirationDate: '2020-06-24T07:00:00+00:00',
-      status: 'APPROVED'
-    }
+const mockNrData = {
+  nrNumber: 'NR 1234567',
+  entityType: 'BEN',
+  filingId: null,
+  applicant: {
+    addressLine1: '45 Frasier Drive',
+    addressLine2: null,
+    addressLine3: null,
+    city: 'Victoria',
+    countryTypeCode: 'CA',
+    postalCode: 'V9E 2A1',
+    stateProvinceCode: 'BC',
+    emailAddress: 'test@gov.bc.ca',
+    phoneNumber: '250-356-9090',
+    firstName: 'John',
+    middleName: 'Joe',
+    lastName: 'Doe'
+  },
+  details: {
+    approvedName: 'MADRONA BREAD BASKET INC.',
+    consentFlag: null,
+    expirationDate: '2020-06-24T07:00:00+00:00',
+    status: 'APPROVED'
   }
+}
```

Comment:
I moved this out of the Describe block so that it can be used by a new Describe section I added below.